src/posit/connect/external/__init__.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+"""
+NOTE: The APIs in this module are provided as a convenience and are subject to breaking changes.
+"""
+
+def _is_local() -> bool:
+    """Returns true if called from a piece of content running on a Connect server.
+
+    The connect server will always set the environment variable `RSTUDIO_PRODUCT=CONNECT`.
+    We can use this environment variable to determine if the content is running locally
+    or on a Connect server.
+    """
+    return not os.getenv("RSTUDIO_PRODUCT") == "CONNECT"
```

Comment:
I haven't found a clear answer to best practices, but I've recently started reducing __init__.py to only include named exports. Then placing any shared module code in a file of the same name; `external/external.py` here.

This also eliminates the need to mark the method as private since it won't be declared in the __init__.py file.